### PR TITLE
Fixing the azure powershell task to take script paths with spaces in it

### PR DESCRIPTION
**Pull request by pryenugu**
- Fixing the azure powershell task to now take script paths with spaces in it

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
Did we manually test both scenarios?
How did you test?

File: Tasks/AzurePowerShell/RunAzurePowerShell.ps1 [hunk 1]
```diff
@@ -17,7 +17,7 @@ Write-Host "Entering script RunAzurePowerShell.ps1"
 Write-Host "ScriptArguments= " $ScriptArguments
 Write-Host "ScriptPath= " $ScriptPath
 
-$scriptCommand = "$ScriptPath $scriptArguments"
+$scriptCommand = "& `"$ScriptPath`" $scriptArguments"
```
Comment (chrisrpatterson) [inline Tasks/AzurePowerShell/RunAzurePowerShell.ps1]: Not sure we need the & here since we are using Invoke-Expression below

**Comment by pryenugu** [comment]
HI I have tested the scenarios manually. Not tested using any automated tests. I observed this issue while implementing the chef task, so applied the same fix to this task as well.

File: Tasks/AzurePowerShell/RunAzurePowerShell.ps1 (hunk 1 above)
Comment (pryenugu) [inline Tasks/AzurePowerShell/RunAzurePowerShell.ps1]: We would need the & as well in this case. It doesn't work without that.

**Pull request merged**